locales/ja-JP.yml

```diff
@@ -976,6 +977,12 @@ _wordMute:
   hard: "ハード"
   mutedNotes: "ミュートされたノート"
 
+_instanceMute:
+  instanceMuteDescription: "ミュートしましたインスタンスのユーザーの返信を含めて設定しましたインスタンスの全てのノートかRenoteをミュートします。"
```

Comment:
```suggestion
  instanceMuteDescription: "ミュートしたインスタンスのユーザーからの返信を含めて、設定したインスタンスの全てのノートとRenoteをミュートします。"```

Reply:
It's probably no problem, but I could be wrong.

Reply:
You have extra backticks at the end of your suggestion, these would break the document. Rest seems ok.

Reply:
Thanks for noticing. I've fixed it.